tests/jeos/firstrun.pm

```diff
@@ -33,21 +34,12 @@ sub select_keyboard {
 }
 
 sub run {
-    if (check_var('VERSION', '12')) {
-        # JeOS-SLE
-        accept_license;
-        select_keyboard;
-        select_locale;
-    }
-    else {
-        # JeOS-SLE-SP1, Leap 42.1
-        select_locale;
-        accept_license if check_var('DISTRI', 'sle');
-        select_keyboard;
-    }
+    select_locale;
+    accept_license if is_sle;
```

Comment:
The order of the calls is different in the old code. I guess it's not relevant anymore, but I wonder why this if-else block was written in this way =)

Reply:
I guess someone hoped for supporting SLE12- and SLE12SP1-based JeOS. Which turned out not to be necessary as there are no new builds after GM. So, now we need just the `else` branch.